overlay/overlay_launchers.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2005-2016 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_OVERLAY_LAUNCHERS_H_
+#define MUMBLE_OVERLAY_LAUNCHERS_H_
+
+static const char *overlayLaunchers[] = {
+	"Steam.exe", // Steam
+	"EALaunchHelper.exe", // Origin
```

Comment:
For me, it is EALaunchHelper.exe still:

![image](https://cloud.githubusercontent.com/assets/36527/23826920/d77fdd66-06a7-11e7-8ccb-ff6114a41403.png)